src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private String symbol;
+    private Person person;
+    private Tag tag;
+    
+    public Tagging(String symbol, Tag tag, Person person) {
```

Comment:
String is not a goof type to maintain the tag operation. Consider using a boolean or a enum
If not this is a good attempt.